bucket/mednafen.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.24.1",
+    "homepage": "https://mednafen.github.io/",
+    "description": "Emulator for different systems",
+    "license": "GNU GPLv2",
```

Comment:
```suggestion
    "description": "Various systems emulator",
    "homepage": "https://mednafen.github.io/",
    "license": "GPL-2.0-only",
```

Reply:
On the license side, I can see that it's probably better in specifying it. But as for the other changes (the order of the homepage and description), even though as I'll merge/commit them anyways, I'd like to know if you have an explanation for it. I supposed it's how all the other packages are written?

Reply:
- Version is required property and main identifier of application. Should be always first.
- Description is the second most valuable information for user who investigate what he will install.
- THen if user wants to know more or the description is not enough, he would go to webpage.
- And finally as soon as he decide to install such application he needs to care about license
	- Corporate statement about some license types
	- License purchase for commercial usage